### PR TITLE
Add LTI session which is used for student views

### PR DESCRIPTION
**Pull request by idegtiarov**
Add LTI session which is used for student views
Improve response xml for grade callback function
Add checking for LTI session breaks which lead to updating activity in un-submitted sequence item

### CODE REVIEW

File: bridge_adaptivity/config/settings/local.py [hunk 1]
```diff
@@ -5,12 +5,12 @@
...
-MIDDLEWARE_CLASSES += (
-    'djdev_panel.middleware.DebugMiddleware',
-    'debug_toolbar.middleware.DebugToolbarMiddleware',
-)
+# INSTALLED_APPS += ('debug_toolbar', 'sslserver')
+#
+# MIDDLEWARE_CLASSES += (
+#     'djdev_panel.middleware.DebugMiddleware',
+#     'debug_toolbar.middleware.DebugToolbarMiddleware',
+# )
```
Comment (wowkalucky) [inline bridge_adaptivity/config/settings/local.py]: git update-index --assume-unchanged local.py

File: bridge_adaptivity/module/utils.py [hunk 2]
```diff
@@ -59,4 +92,19 @@ def parse_callback_grade_xml(body):
...
         raise BridgeError('score value outside the permitted range of 0.0-1.0')
 
-    return sourced_id, score
+    return imsx_message_identifier, sourced_id, score, action
+
+
+def chose_activity(sequence_item, sequence=None):
+    if sequence_item is None:
+        sequence = sequence
+    else:
```
Comment (wowkalucky) [inline bridge_adaptivity/module/utils.py]: Do we really need increase complexity here?

File: bridge_adaptivity/module/views.py [hunk 3]
```diff
@@ -146,33 +150,34 @@ def sequence_item_next(request, pk):
...
-        next_sequence_item = SequenceItem.objects.create(
-            sequence=sequence_item.sequence,
-            activity=activity,
-            position=sequence_item.position + 1
-        )
-
+    log.debug("Picked next sequence item is: {}".format(next_sequence_item))
+
+    if not next_sequence_item or next_sequence_item.position == last_item:
+        activity = utils.chose_activity(sequence_item)
```
Comment (wowkalucky) [inline bridge_adaptivity/module/views.py]: It looks like this `if` can be inverted for the sake of sanity...

File: bridge_adaptivity/config/settings/local.py (hunk 1 above)
Comment (idegtiarov) [reply bridge_adaptivity/config/settings/local.py]: 👍

File: bridge_adaptivity/module/utils.py (hunk 2 above)
Comment (idegtiarov) [reply bridge_adaptivity/module/utils.py]: Actually, we have no choice and have to decide which sequence should be used, but I will rewrite this.

File: bridge_adaptivity/module/views.py (hunk 3 above)
Comment (idegtiarov) [reply bridge_adaptivity/module/views.py]: sure, nice catch!

**Pull request merged**